lib/pipl/default.rb

```diff
@@ -65,6 +65,10 @@ def top_match
         ENV['PIPL_TOP_MATCH']
       end
 
+      def api_endpoint
+        ENV.fetch 'PIPL_SEARCH_API_ENDPOINT', API_ENDPOINT
```

Comment:
i think we should use env[...] here
https://stackoverflow.com/questions/47985660/rails-difference-between-env-fetch-and-env